src/test/java/com/mycodefu/vsssfshss/names/NameGeneratorTest.java

```diff
@@ -0,0 +1,32 @@
+package com.mycodefu.vsssfshss.names;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class NameGeneratorTest {
+    @Test
+    void generateName() {
+        String name = NameGenerator.generateName();
+        System.out.println(name);
+
+        assertNotNull(name);
+        assertTrue(name.length() > 2);
+    }
+
+    @Test
+    void generateUniqueNames() {
+        List<String> nameList = IntStream.range(0, 100000).mapToObj(value -> NameGenerator.generateName()).toList();
+
+        for (int i = 0; i < 1000; i++) {
+            String name = NameGenerator.generateName();
+            if (nameList.contains(name)) {
```

Comment:
This should probably be removed sinse the next line (assertFalse) accomplishes the same thing, and doing it twice does not really add anything useful.

Reply:
Done